text/0031-automatic-backups.md

```diff
@@ -0,0 +1,99 @@
+# Summary
+
+Add built-in backup functionality to OBS Studio.
+
+# Motivation
+
+As of writing OBS does not offer any native option for users to have their settings (profiles and scene collections)
+automatically backed up to cloud, local, or network storage. At the moment backing up OBS' entire configuration data
+is also not easily possible or intuitively explained for non-technical users.
+
+This RFC is meant to firstly facilitate a discussion to determine a high-level approach to implementing a backup
+feature in OBS Studio, before then formulating a specification for the low-level implementation of the chosen approach.
+
+# Detailed changes
+
+To start, I propose the following three approaches for discussion (though more should be suggested and discussed):
+
+ 1. Automatic export to user-specified storage location
+ 2. Integration of third party (cloud) storage providers
+ 3. Backup infrastructure and account system operated by the OBS Project
+
+Additionally, this RFC will lay out some ideas for how to integrate these backup/restore functionality into the OBS UI.
+
+## Overview of the three suggested variants
+
+### 1. Automatic export
+
+Arguably the easiest approach is to add an option to automatically export/save OBS configuration files to an additional
+location when they are written. This could be configured by the user to be a synced folder (e.g. Dropbox or Google Drive)
+or a network drive, but could also be simply a different storage drive for redundancy.
+
+Versioned backups could be implemented by allowing backups to be sorted by date or time in the filename or folder path.
```

Comment:
If it's a single file, just naming them with a datetime convention would do the trick.
If you do go down the path of having assets (optionally?) backed up as well, then would using Git be a better choice?